pkg/cosign/cosign.go

```diff
@@ -261,8 +261,8 @@ func FetchAttestations(ctx context.Context, rclient registryclient.Client, opts
 
 	signatures, bundleVerified, err := tracing.ChildSpan3(
 		ctx,
-		"cosign_operations",
-		"verify_image_signatures",
+		"",
+		"VERIFY IMG ATTS",
```

Comment:
What does "ATTS" mean? Attestations or attestors?

Reply:
Attestations, shall I make it more clear in the operation name ?

Reply:
fixed